eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -8,8 +8,8 @@ parameters:
   ScriptDirectory: eng/common/scripts
   TargetDocRepoName: ''
   TargetDocRepoOwner: ''
-  PRBranchName: 'smoke-test-rdme'
-  SourceBranchName: 'smoke-test'
+  PRBranchName: 'master-rdme'
+  SourceBranchName: 'master'
   PRLabels: 'auto-merge'
```

Comment:
As we were chatting yesterday this auto-merge label isn't being passed through to the second create-pull-request at the bottom of this file.

Reply:
Yessir. I'll do a follow-up.